Microsoft.Toolkit.Uwp.UI.Controls/RotatorTile/RotatorTile.cs

```diff
@@ -244,12 +244,11 @@ private void RotateToNextItem()
 
                 if (_currentElement != null)
                 {
-                    _currentElement.DataContext = _nextElement.DataContext;
-                }
-
-                if (_nextElement != null)
-                {
-                    _nextElement.DataContext = GetNext(); // Preload the next tile
+                    if (_nextElement != null)
```

Comment:
you could merge this with the _currentElement check ex if(_currentElement != null && _nextElement != null)